MdeModulePkg/Universal/Console/ConSplitterDxe/ConSplitter.c

```diff
@@ -5001,9 +5020,11 @@ ConSplitterTextOutClearScreen (
   // been checked in ConSplitterTextOutSetCursorPosition. And (0, 0) should
   // always be supported.
   //
-  Private->TextOutMode.CursorColumn  = 0;
-  Private->TextOutMode.CursorRow     = 0;
-  Private->TextOutMode.CursorVisible = TRUE;
+  Private->TextOutMode.CursorColumn = 0;
+  Private->TextOutMode.CursorRow    = 0;
+  // MS_CHANGE_?
+  // MSCHANGE - Default to cursor not visible
+  Private->TextOutMode.CursorVisible = FALSE;

```

Comment:
This can be single line, with comment to right, don't need to change spacing for other two.